cachecontrol/heuristics.py

```diff
@@ -77,3 +78,36 @@ def update_headers(self, response):
     def warning(self, response):
         tmpl = '110 - Automatically cached for %s. Response might be stale'
         return tmpl % self.delta
+
+_ONE_DAY_IN_SECONDS = timedelta(days=1)
+
+class LastModifiedHeuristic(BaseHeuristic):
+    """
+    Apply the heuristic suggested by RFC 7234, 4.2.2,
+    using the Last-Modified date, when no explicit
+    freshness is specified.
+    """
+
+    def update_headers(self, response):
+        if 'expires' not in response.headers:
+            if 'cache-control' not in response.headers or response.headers['cache-control'] == 'public':
+                # RFC 7231, 6.1
+                if response.status in [200, 203, 204, 206, 300, 301, 404, 405, 410, 414, 501]:
```

Comment:
Could we move this list of status codes to be an attribute and make it a set? That will make it a tiny bit more performant and then you can give the status codes a better name and potentially remove the comment.